qiskit/circuit/instruction.py

```diff
@@ -103,6 +105,38 @@ def __init__(self, name, num_qubits, num_clbits, params, duration=None, unit="dt
 
         self.params = params  # must be at last (other properties may be required for validation)
 
+    @property
+    def base_class(self) -> Type[Instruction]:
+        """Get the base class of this instruction.  This is guaranteed to be in the inheritance tree
+        of ``self``.
+
+        The "base class" of an instruction is the lowest class in its inheritance tree that the
+        object should be considered entirely compatible with for _all_ circuit applications.  This
+        typically means that the subclass is defined purely to offer some sort of programmer
+        convenience over the base class, and the base class is the "true" class for a behavioural
+        perspective.  In particular, you should *not* override :attr:`base_class` if you are
```

Comment:
IIRC bold is double, italic is single:

```suggestion
        perspective.  In particular, you should **not** override :attr:`base_class` if you are
```

Reply:
That's right yeah, but I was aiming for italics haha - I don't ever really use bold text.  If we typically do in our style, though, I'll change it.